### ISSUE TITLE
session error after installing in a working cluster

### ISSUE THREAD

**Issue by jalberto** [sivasamyk/logtrail#190]
in a ES 5.6.2 working cluster, installing logtrail-5.6.2 in k8s 1.7.2

![screenshot from 2017-10-05 11-28-16](https://user-images.githubusercontent.com/99927/31220395-96d69c84-a9c0-11e7-94a9-46d8260a66c7.png)

**Comment by sivasamyk** [comment]
This looks similar to this issue : https://github.com/elastic/kibana/issues/9369

Recommendation : Delete `optimize/bundles` and restart Kibana. There are other workaround suggested here.

**Comment by jalberto** [comment]
yep, already tried those workarounds, didn't work.
notice I am on kubernetes, so I cannot "reboot node" I recreated it, anyway didn't work.

**Comment by sivasamyk** [comment]
Do you have x-pack installed? Do you have multiple instances of Kibana running behind K8S service? If x-pack is installed can you try disabling security by `xpack.security.enabled: false` in elasticsearch.yml and kibana.yml?

**Comment by jalberto** [comment]
yes, x-pack is installed, but every premium feature is disabled (in ES and kibana).
ES is a cluster, but kibana only have one instance

**Comment by sivasamyk** [comment]
How do you deploy logtrail plugin in Kibana docker? Is it pre-installed during docker build or it is installed during bring up of pod in k8s? If you can build custom kibana docker images, can you have Kibana & Logtrail installed in local machine and get it working. Build a docker image based on this working setup (can copy the whole dir) by writing a custom Dockerfile. I do not have access to k8s and x-pack to do the above setup.

**Comment by jalberto** [comment]
I tried several methods:

1. using postStart hook -> got reported error
2. using a shell though kubectl exec -> same erro
3. building a custom image -> works as expected

So maybe add to documentation this bit is important meanwhile there is a way to install logtrail in live kibana

**Comment by sivasamyk** [comment]
Plugin installation requires restart of Kibana. This is documented in Logtrail Readme.

**Comment by jalberto** [comment]
yes, but even restart doesn't works, at least in k8s context as the most similar thing is recreate pod

**Comment by sivasamyk** [comment]
When you using postStart hook, logtrail will be installed after start of Kibana. Even recreate of pod will result in same. Installing the logtrail as part of docker build is the solution.

**Comment by jalberto** [comment]
installing  by hand using `kubectl exec` end then recreating the pod (using persistent volumes of course) doesn't works either

**Comment by sivasamyk** [comment]
Hmm. There is some state which we are losing while creating new pod. Custom image is the only solution then.

**Comment by jalberto** [comment]
looks like, for that reason I suggest to explicitly mention it in the docs, so next one dont spent too much time "fixing" this :)
